src/amo/components/AddonReview.js

```diff
@@ -74,31 +84,38 @@ export class AddonReviewBase extends React.Component {
       );
     }
 
+    const titlePlaceholder = i18n.gettext('Give your review a title');
+
     return (
-      <div className="AddonReview">
+      <OverlayCard ref={(ref) => { this.overlayCard = ref; }}
+        visibleOnLoad className="AddonReview">
         <h2 className="AddonReview-header">{i18n.gettext('Write a review')}</h2>
         <p ref={(ref) => { this.reviewPrompt = ref; }}>{prompt}</p>
         <form onSubmit={this.onSubmit} ref={(ref) => { this.reviewForm = ref; }}>
-          <textarea
-            className="AddonReview-textarea"
-            ref={(ref) => { this.reviewTextarea = ref; }}
-            name="review"
-            placeholder={placeholder}>
-            {review.body}
-          </textarea>
-          <div className="AddonReview-button-row">
-            <button className="AddonReview-button AddonReview-back-button"
-              onClick={this.goBackToAddonDetail}
-              ref={(ref) => { this.backButton = ref; }}>
-              {i18n.gettext('Back')}
-            </button>
+          <div className="AddonReview-form-input">
+            {errorHandler.getError()}
             <input
-              className="AddonReview-button AddonReview-submit-button"
-              type="submit" value={i18n.gettext('Submit')}
-            />
+              ref={(ref) => { this.reviewTitleInput = ref; }}
+              type="text"
+              className="AddonReview-title"
+              name="reviewTitle"
+              value={reviewTitle}
+              onInput={this.onTitleInput}
+              placeholder={titlePlaceholder} />
+            <textarea
```

Comment:
When there are more than 4 properties, I find them easier to scan if each is on a new line. I can balance the closing tags on these, that looks a bit better.